### DISCUSSION TITLE
Can't search any of my private repositories

### DISCUSSION THREAD

**Discussion by VinayBanakar** [community/community#15620] [Other Feature Feedback, Questions, & Ideas] [answered]
In Github Code Search preview,
**Why wasn't my code found?**
>The index currently includes more than 7 million popular public repositories, plus all private content accessible to members of the technical preview.

Does this mean all private repositories of the preview members are searchable? if so, I am not able to find mine.
I get `Unkown repo:VinayBanakar/XYZ; Query is not satisfiable` error. 

Thanks!

**Comment by colinwm** [comment, marked as answer]
Hey @VinayBanakar, your private code should be indexed, although it can take a few minutes after you create a new repository. Is the repository new? Can you give it another try and see if is available now?
